include/net/nrf_cloud_rest.h

```diff
@@ -67,7 +67,9 @@ struct nrf_cloud_rest_context {
 	/** Timeout value, in milliseconds, for REST request. The timeout is set individually
 	 * for socket connection creation and data transfer meaning REST request can take
 	 * longer than this given timeout.
-	 * For no timeout, set to NRF_CLOUD_REST_TIMEOUT_NONE.
+	 * For no timeout (wait forever) set to less than or equal to zero,
+	 * or NRF_CLOUD_REST_TIMEOUT_NONE.
```

Comment:
If we stick to the use of `SYS_FOREVER_MS`, I would probably even remove `NRF_CLOUD_REST_TIMEOUT_NONE` and instruct in this comment to use just `SYS_FOREVER_MS` for indefinite timeout.

Reply:
i would like to keep the values and their behavior documented so users know what behavior to expect.